### DISCUSSION TITLE
IANA media type for UWLSWD HTML+RDFa

### DISCUSSION THREAD

**Discussion by briesenberg07** [uwlib-cams/uwlswd#75] [technical]
# what the the IANA media type for UWLSWD HTML+RDFa files?

## REFERENCE
- [Media Types](https://www.iana.org/assignments/media-types/media-types.xhtml)
- [20230908_media_type_email_thread.pdf](https://github.com/uwlib-cams/uwlswd/files/13517313/20230908_media_type_email_thread.pdf)

--------------------------------

[DataCite schema documentation](https://schema.datacite.org/meta/kernel-4.4/doc/DataCite-MetadataKernel_v4.4.pdf) has:

> Use file extension or MIME type where possible, e.g., PDF, XML, MPG or application/pdf, text/xml, video/mpeg. 

Per [discussion](https://docs.google.com/document/d/13OK-OqBnqaihnfByu-wZa65F0AlpVTrUYHWZs2SZAUk/edit#bookmark=id.9yxu3d57e65k), we have decided to use the IANA Media Type `text/html`

TO DO: Confirm updated values in DataCite XML

- [x] uwlib.55.a.3.1.xml (no format)
- [x] uwlib.55.a.3.2.xml (no format)
- [x] uwlib.55.a.3.3.xml (no format)
- [x] uwlib.55.a.3.4.xml (no format)
- [x] uwlib.55.a.3.5.xml (no format)
- [x] uwlib.55.a.3.6.xml (no format)
- [ ] uwlib.55.a.xml (incorrect format)
- [x] uwlib.55.b.1.xml (incorrect format)
- [x] uwlib.55.b.2.xml
- [x] uwlib.55.b.3.xml
- [x] uwlib.55.d.1.xml (incorrect format)
- [x] uwlib.55.d.2.xml
- [x] uwlib.55.d.3.xml (incorrect format)
- [x] uwlib.55.d.4.xml (no format)
- [ ] uwlib.55.d.5.xml (no format)
- [x] uwlswd.06xx-6744.xml (gives all formats\*, has incorrect `application/xhtml+xml`)
- [x] uwlswd.084w-3778.xml (gives all formats\*, has incorrect `application/xhtml+xml`)
- [x] uwlswd.1c2x-cj09.xml (gives all formats\*, has incorrect `application/xhtml+xml`)
- [x] uwlswd.23tq-5e25.xml (gives all formats\*, has incorrect `application/xhtml+xml`)
- [x] uwlswd.25z4-e963.xml (no formats)
- [x] uwlswd.2e2b-y833.xml (no formats)
- [x] uwlswd.2eg3-1x53.xml (...)
- [x] uwlswd.3d5s-zx23.xml
- [x] uwlswd.42d2-ca58.xml
- [x] uwlswd.4a8z-tq62.xml
- [x] uwlswd.4jrs-m847.xml
- [x] uwlswd.62zz-1534.xml
- [x] uwlswd.633m-h220.xml
- [x] uwlswd.8rpj-ek77.xml
- [x] uwlswd.aec4-nv40.xml
- [x] uwlswd.axz0-z371.xml
- [x] uwlswd.dh5m-5y16.xml
- [x] uwlswd.eje7-jq11.xml
- [x] uwlswd.f447-ax91.xml
- [x] uwlswd.ggnh-4s58.xml
- [x] uwlswd.gq3z-mv97.xml
- [x] uwlswd.jfr5-z647.xml
- [x] uwlswd.mkjn-bp10.xml
- [x] uwlswd.npns-e903.xml
- [x] uwlswd.nt0v-d633.xml
- [x] uwlswd.qgc0-9r48.xml
- [x] uwlswd.rw25-p125.xml
- [x] uwlswd.vw5v-gh79.xml
- [x] uwlswd.x4ce-sd21.xml
- [x] uwlswd.ypcx-6882.xml
- [x] uwlswd.ywjs-vr46.xml

**Comment by cspayne** [comment]
This has been adjusted in rdf2datacite.xsl, but datacite metadata files will still need to be updated

**Comment by briesenberg07** [comment]
Update the following DataCite XML files-- add or correct `formats/format`

\* @gerontakos I note that MARC 00x DOI metadata gives all formats. I thought that for DOI, would would only give *one* format -- `text/html` -- for the resource where the DOI will dereference. 

Do you think we should list all formats in DataCite XML?

**Comment by gerontakos** [comment]
I think the inconsistency is the big problem. However, yes, I think all formats makes most sense in the DataCite XML. Thinking of the user in DataCite: "I do want that dataset of concepts for music formats, but I want it in turtle." If the data says it's only available as HTML (the format retrieved), that user's eyes may fill with tears, seeing it is only available as RDFa. However, from the very page retrieved, there is a link to the desired turtle! Amazing! The user will be thrilled, and would probably like to know while searching and retrieving the DataCite metadata.

You know, this is one of the things FRBR Group 1 entities resolves: The retrieved page describes the dataset work and offers avenues to the different expressions. But, in our record-laden xml Datacite record, what are we describing? Lots of things! So how do we keep it straight?

Anyway, I could go either way. Consistency is the thing. What we decide for one, we should do for all. But, even then, it's the Datacite. We don't need to change things retrospectively every time we change our mind.

**Comment by cspayne** [comment]
**DECISION MADE - 2023-10-26 UWLSWD meeting:**
 DataCite metadata shall have *one* formats/format value, `text/html`
